canonical-web-and-design/ubuntu.com
# Add script to help load releases

## Done
- Add support and esm expiry dates to the release table.
- Add a little script to help us have initial releases data for ubuntu security features.

## QA

- dotrun clean
- dotrun exec alembic upgrade head
- dotrun exec python3 webapp/security/fixtures/releases.py
- dotrun
- check `/security/notices` and see if releases appear in the dropdown.
- They should appear from newest to oldest.
